stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/images/CoilStreamImageLoader.kt

```diff
@@ -26,8 +26,7 @@ import android.os.Build
 import android.widget.ImageView
 import coil.drawable.MovieDrawable
 import coil.drawable.ScaleDrawable
-import coil.fetch.VideoFrameUriFetcher
-import coil.loadAny
+import coil.load
```

Comment:
So they renamed `loadAny` to `load`, which forces us to swap the names. 😂

Reply:
Done: [585cc0d](https://github.com/GetStream/stream-chat-android/pull/3538/commits/585cc0d4bf017cdf797ddc7f433a2108c2434cba)